docs/upgrading.rst

```diff
@@ -149,3 +149,51 @@ Re-use of environments
 
     [testenv:b]
     deps = pytest<7
+
+
+Legacy CLI entry point may result in ambiguous usages
```

Comment:
Probably should be shorter: CLI compatibility is enough.

Reply:
There's another section on "CLI arguments", so I called it "CLI command compatibility", but we could also combine the sections later maybe.